platform/qemu.go

```diff
@@ -572,7 +572,8 @@ func (builder *QemuBuilder) Exec() (*QemuInstance, error) {
 			"--ctrl", fmt.Sprintf("type=unixio,path=%s", swtpmSock),
 			"--terminate", "--tpmstate", fmt.Sprintf("dir=%s", inst.swtpmTmpd))
 		cmd := inst.swtpm.(*exec.ExecCmd)
-		cmd.Stderr = os.Stderr
+		// For now silence the swtpm stderr as it prints errors when
+		// disconnected, but that's normal.
```

Comment:
Could we toggle it based on `--verbose`? That way, we can still easily inspect it if something keeps failing.

Reply:
Looked at that but the options aren't plumbed through, and that `--verbose` already defaults to true (oddly).  I don't think it's worth the time on this.